### ISSUE TITLE
Field Type: Group?

### ISSUE THREAD

**Issue by lautiamkok** [htmlburger/carbon-fields#1010]
I am trying to migrate from AFC and I use the Group field type quite often with AFC. But I can't find this field type in Carbon Fields.

I know there is a Complex group type which is a Repeater in AFC but it is not the same as the  Group field type which only groups a range to other field types.

Any solution for this?

**Comment by JuanBernal98** [comment]
Hi!
Tabs allow you to group multiple fields under different tab panels using the Container method
https://carbonfields.net/docs/containers-tabs/

**Comment by lautiamkok** [comment]
@JuanBernal98 yes, tabs provide visual grouping, but it has no data like below:

```
array(
 'filter' => array(
   'colour => 'xxx',
   'opacity' => '0.4'
  )
)
```

**Comment by devi-vahid** [comment]
> 
> 
> @JuanBernal98 yes, tabs provide visual grouping, but they have no data like below:
> 
> ```
> array(
>  'filter' => array(
>    'colour => 'xxx',
>    'opacity' => '0.4'
>   )
> )
> ```

yes that's exactly what I'm looking for, ended up reading this question
I want to save multiple input field values in the same database row as an array, is there a way to do this with Carbon Fields?

**Comment by lautiamkok** [comment]
@devi-vahid there is a Group field type already but it seems that no way to use it like other field types. It is tied up with the Complex field type.

So I ended up changing my data structure by making all my group data repeatable. It can be useful even though some of them would never need that.

**Comment by devi-vahid** [comment]
@lautiamkok thanks for your reply, i did the same but you can set the [`set_max( 1 )`](https://docs.carbonfields.net/learn/fields/complex.html#config-methods-2) so users can't add more than one row to the group.

**Comment by lautiamkok** [comment]
Cheers @devi-vahid. Yeah, using `set_max( 1 )` is a good idea to get around this issue too! :-D
